.github/main.workflow

```diff
@@ -51,7 +51,7 @@ action "Test coverage" {
   env = {
     PYTHON_VERSION = "3.7"
     MIN_COVERAGE = "100"
-    PIP_PACKAGES = "coverage pytest pytest-cov .[dev]"
+    PIP_PACKAGES = "coverage pytest pytest-cov .[tests]"
```

Comment:
I would guess you could just do `    PIP_PACKAGES = "coverage pytest pytest-cov .[tests]"` as `coverage` `pytest` and `pytest-cov` are dependencies of `tests`

Reply:
will do